collectors/bzimport/fixups.py

```diff
@@ -171,13 +171,13 @@ def fix_title(self) -> None:
         we strip the summary prefixes without checking them
         and then try to get and store the component
         """
-        if "summary" not in self.flaw_json:
+        if "cve_description" not in self.flaw_json:
```

Comment:
I think `flaw_json` contains the Flaw data obtained from Bugzilla and thus there will be still `summary`.